### DISCUSSION TITLE
Stripe Purchases data / information

### DISCUSSION THREAD

**Discussion by jxs714** [pretix/pretix#4117] [Q&A] [answered]
Hey guys,

I am trying to get full data transferred from Pretix to Stripe such as description

For some reason, description shows it as EVENTNAME-ORDERNUMBER. There is no metadata information such as the ticket name, name, source title etc. Is there anyway I can get this information into Stripe so that I can easily track the sales of the business / name, ticket name, etc.

Please advise, thanks.

**Comment by raphaelm** [comment, marked as answer]
No that is not implemented/supported
